### PR TITLE
Update: `BinanceTech.Binance` version 1.19.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20927)

### CODE REVIEW

File: manifests/b/BinanceTech/Binance/1.19.0/BinanceTech.Binance.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+#  Scope:
```
Comment (OfficialEsco) [inline manifests/b/BinanceTech/Binance/1.19.0/BinanceTech.Binance.installer.yaml:15]:
```suggestion
  Scope: machine
```

File: manifests/b/BinanceTech/Binance/1.19.0/BinanceTech.Binance.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+PublisherUrl: https://www.binance.com/en
+PublisherSupportUrl: https://www.binance.com/en/support-center
+PrivacyUrl: https://www.binance.com/en/privacy
+#Author: 
+PackageName: Binance
+PackageUrl: https://www.binance.com/en
+License: Copyright © 2021 Binance, Inc.
+#LicenseUrl: 
+#Copyright: 
+#CopyrightUrl:
```
Comment (OfficialEsco) [inline manifests/b/BinanceTech/Binance/1.19.0/BinanceTech.Binance.locale.en-US.yaml:17]:
```suggestion
License: Proprietary
LicenseUrl: https://www.binance.com/en/terms
#Copyright: Copyright © 2021 Binance, Inc.
CopyrightUrl: https://www.binance.com/en/terms
```

File: manifests/b/BinanceTech/Binance/1.19.0/BinanceTech.Binance.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+PackageIdentifier: BinanceTech.Binance
+PackageVersion: 1.19.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (OfficialEsco) [inline manifests/b/BinanceTech/Binance/1.19.0/BinanceTech.Binance.installer.yaml:13]:
```suggestion
  Architecture: x64
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Think the issue is the `Architecture`

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**